cypress/integration/home-work3.spec.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+
+import {HomePo} from "../support/home.po";
+import {GiftCardPo} from "../support/gift-card.po";
+import {GiftCatdProductPo} from "../support/gift-catd-product.po";
+import {GiftProductHucPo} from "../support/gift-product-huc.po";
+
+describe('check gift cards', () => {
+
+    const homePage = new HomePo();
+    const giftCardPage = new GiftCardPo();
+    const giftProdHucPage = new GiftProductHucPo();
+    const giftCardProductPage = new GiftCatdProductPo();
+
+    beforeEach(() => {
+        homePage.navigateTo();
+    });
+
+    it('check set gift amount', () => {
+        const successAddedMessage = 'Added to Cart';
+        const giftPrice = '$100.00';
+
+        homePage.clickOnElem(homePage.secondaryNavElem.giftCardsLink);
+
+        // verify that user redirected to page with route /gift-cards/
+        homePage.verifyUrl(giftCardPage.pageUrl);
+
+        cy.get(giftCardPage.giftCardsLinks).as('giftCardArray');
+        cy.get('@giftCardArray').each(($giftCard) => {
+            // I want to check that all cards are available with something like element.toBeClickable, but I can't
+            // find this in Cypress, I know that before we make any action with the element it will be checked by
+            // default, but I think it is better to know that all my cards can be choose before action with any of them.
+            // How can I check that all cards are available?
```

Comment:
cy.wrap($giftCard)
            .should('be.visible')
            .and('not.be.disabled')
            .and('not.have.focus')

Reply:
And you're right - it's not necessary to check that element is clickable, of you'll click on it just after check.